functions/src/routers/groupRouter.ts

```diff
@@ -49,36 +87,126 @@ groupRouter.post("/add", async (req, res) => {
     });
   }
 
-  const groupID=englishName.trim().toLowerCase()
-  // TODO: memberCount 및 members 정의 후에 해당 요청이 문제가 없을경우 
+  const groupID = englishName.trim().toLowerCase();
+  // TODO: memberCount 및 members 정의 후에 해당 요청이 문제가 없을경우
   // 해당 유저의 정보에 groups 항목에 해당 그룹을 추가한다.
   admin
     .firestore()
-    .collection("group")
+    .collection("groups")
     .doc(groupID)
     .set({
       name: name,
       type: type,
       description: description,
       owners: [uid],
       members: [uid],
-      memberCount: 1 // memberCount 그룹 추가 시 1명이므로
-    }).then(async ()=>{
+      memberCount: 1, // memberCount 그룹 추가 시 1명이므로
+    })
+    .then(async () => {
       //문제가 없음
       try {
-        await admin.firestore().collection("users").doc(uid).update({
-          groups: admin.firestore.FieldValue.arrayUnion(groupID)
-        })
-        return res.send("Success")
-      }catch(e) {
-        admin.firestore().collection("group").doc(groupID).delete()
-        return res.status(500).json(e)
+        await admin
+          .firestore()
+          .collection("users")
+          .doc(uid)
+          .update({
+            groups: admin.firestore.FieldValue.arrayUnion(groupID),
+          });
+        return res.send("Success");
+      } catch (e) {
+        admin.firestore().collection("group").doc(groupID).delete();
+        return res.status(500).json(e);
       }
-    })
-  
-    // if(!memberCount){
-    // admin.firestore().collection("users").doc(uid).set({...createdDocument, group: name});
-    // }
+    });
+
+  // if(!memberCount){
+  // admin.firestore().collection("users").doc(uid).set({...createdDocument, group: name});
+  // }
   return;
 });
+groupRouter.get("/:userId/groups", async (req, res) => {
+  const uid = req.decodedToken.uid;
+
+  try {
+    // 데이터베이스에서 해당 유저의 정보 가져오기
+    const userData = (
+      await admin.firestore().collection("users").doc(uid).get()
+    ).data();
+
+    // console.log(uid, userData);
+
+    // DB 에서 해당 데이터를 찾을 수 없는경우.
+    if (!userData)
+      return res.status(404).json({ status: 404, error: "USERDATA NOT FOUND" });
+
+    // 해당 유저 그룹에 그룹 정보가 있는지 확인
+    if (!userData.group)
```

Comment:
위에서 말한 것처럼 `users` 정보에는 `groups` 를 이용하셨기때문에 groups 으로 바꿔주세요.

```suggestion
    if (!userData.groups)
```